packages/terra-folder-tree/src/subcomponents/FolderTreeItem.jsx

```diff
@@ -112,6 +123,48 @@ const FolderTreeItem = ({
     }
   };
 
+  const handleKeyDown = event => {
+    switch (event.nativeEvent.keyCode) {
+      case KeyCode.KEY_RETURN:
+      case KeyCode.KEY_SPACE:
+        event.preventDefault();
+
+        if (isFolder) {
+          handleToggle(event);
+        } else {
+          onClick(event);
+        }
+        break;
+      case KeyCode.KEY_LEFT: {
+        event.preventDefault();
+        if (event.metaKey) { break; }
+
+        if (isFolder && isExpanded) {
+          handleToggle(event);
+        } else {
+          parentRef?.current.focus();
```

Comment:
Do we need a parent reference object?  Can't we just use the parentElement of the itemNode?

Reply:
The parent element is of the current node is actually a <ul>. What we need is that ul's sibling which is a <li>. And since there can be multiple siblings, I found this to be the easiest way to get the right li element we need.

Reply:
I am confused.  Wouldn't the list items be li elements within a folder that is a ul element?  Why wouldn't the parent of the list item be its own folder?

Reply:
Not in this folder tree, the folder list item is a separate li element. For example:
<img width="1266" alt="Screenshot 2023-12-18 at 2 10 02 PM" src="https://github.com/cerner/terra-framework/assets/8482248/d9746cce-1dab-4699-a26a-e39ad030ff81">

Document 4 & 5 are both grouped in a ul but the folder it's in, Projects - Level 4, is its own li item

<img width="1263" alt="Screenshot 2023-12-18 at 2 10 40 PM" src="https://github.com/cerner/terra-framework/assets/8482248/60019adf-ad37-43f7-aa98-a88ad1338364">